index.html

```diff
@@ -228,7 +228,8 @@ <h2> Multi-Post Stories</h2>
         <h2>About Myself</h2>
       </div>
       <p>
-        Hello, I'm Gilbert Mutai a software developer! I can help you build a product , feature or website.Look through some of my work
```

Comment:
[ OPTIONAL]
- [ ] Kindly make sure your nav bar fits perfectly fine. `you can use the flexbox`
![image](https://user-images.githubusercontent.com/97100091/193896734-1cff8d41-b79a-4c3c-91de-a97c3feebbed.png)